parse.py

```diff
@@ -83,6 +83,8 @@ def _skip_column(self, key, value):
 
         # Check if val empty
         if pd.isna(value):
+            if key == "associated":
```

Comment:
Add a comment describing the purpose of this conditional.

Reply:
Just added a comment